anker/src/instruction.rs

```diff
@@ -89,11 +89,11 @@ accounts_struct! {
             is_writable: false,
         },
         // Instance of the token swap data used for trading StSOL for UST.
-        pub pool {
+        pub token_swap_pool {
             is_signer: false,
             is_writable: false,
         },
-        pub rewards_destination {
+        pub terra_rewards_destination {
```

Comment:
I think we will eventually need to put this in the payload and not in the accounts, because I think it is going to be a different format pubkey, right? But for now this is fine, we can change it when we get to the Wormhole part.

Reply:
Yeah, I think so. I think it's better to store it in Anker's state so it would be slightly more difficult to change.